contributors/design-proposals/node/cri-rawproc.md

```diff
@@ -0,0 +1,50 @@
+# CRI: RawProc Option
+
+## Background
+
+Currently the way docker and most other container runtimes work is by masking
+and setting as read-only certain paths in `/proc`. This is to prevent data
+from being exposed into a container that should not be. However, there are
+certain use-cases where it is necessary to turn this off.
+
```

Comment:
Instead of adding a `rawproc` option at CRI level, why don't we just expose `maskedPaths` and `readonlyPaths` ([link](https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#masked-paths)) in CRI?

I'm not saying that we should do that, but I want to make sure we've considered that option.

I think it is more generic. If needed, we can leverage that to mask/unmask or set read-only 
for other paths in the future, instead of adding another `rawXXX`.

Reply:
I agree with @Random-Liu. We could just expose maskedPaths and readonlyPaths. 
A higher level abstraction makes more sense when it isn't portable between runtimes.

Reply:
ok thats for cri, but what do you want for the top level, the same? I will do whatever everyone wants, I really have no fight in the naming, I just need a way to do this.